### PR TITLE
Adding tests for `dapr uninstall -k --all`

### LINKED ISSUE #656: Dapr uninstall -k does not remove the CRDs
## Expected Behavior

`dapr uninstall -k` remove the CRDs

## Actual Behavior
```
$ dapr uninstall -k
ℹ️  Removing Dapr from your cluster...
✅  Dapr has been removed successfully
$ kubectl get crds
NAME                     CREATED AT
components.dapr.io       2021-03-31T01:24:00Z
configurations.dapr.io   2021-03-31T01:24:00Z
subscriptions.dapr.io    2021-03-31T01:24:01Z
```

## Steps to Reproduce the Problem

- install dapr using `dapr init -k`. 
- run `kubectl get crds` 
- run `dapr uninstall -k` 
- run `kubectl get crds` 
the crds still exist. 

## Release Note

RELEASE NOTE: **RESOLVED** `dapr uninstall -k` does not remove CRDs

### PR DESCRIPTION
**Pull request by pkedy**
# Description

Adding E2E tests to accompany #656

## Issue reference

Closes #656 (for real)

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: pkg/kubernetes/components.go [hunk 1]
```diff
@@ -35,7 +36,21 @@ func PrintComponents(name, outputFormat string) error {
 			return nil, err
 		}
 
-		return client.ComponentsV1alpha1().Components(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
+		list, err := client.ComponentsV1alpha1().Components(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
+		if err != nil {
+			// This means that the Dapr Components CRD is not installed and there
```
Comment (yaron2) [inline pkg/kubernetes/components.go:41]: Nit: Rephrase comment?

File: pkg/kubernetes/components.go [hunk 2]
```diff
@@ -35,7 +36,21 @@ func PrintComponents(name, outputFormat string) error {
 			return nil, err
 		}
 
-		return client.ComponentsV1alpha1().Components(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
+		list, err := client.ComponentsV1alpha1().Components(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
+		if err != nil {
```
Comment (yaron2) [inline pkg/kubernetes/components.go:40]: Better to use `"k8s.io/apimachinery/pkg/api/errors"` and `errors.IsNotFound(err)`

File: pkg/kubernetes/configurations.go [hunk 3]
```diff
@@ -39,7 +40,21 @@ func PrintConfigurations(name, outputFormat string) error {
 			return nil, err
 		}
 
-		return client.ConfigurationV1alpha1().Configurations(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
+		list, err := client.ConfigurationV1alpha1().Configurations(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
```
Comment (yaron2) [inline pkg/kubernetes/configurations.go:43]: See comment about "k8s.io/apimachinery/pkg/api/errors"

File: pkg/kubernetes/mtls.go [hunk 4]
```diff
@@ -37,14 +39,24 @@ func getSystemConfig() (*v1alpha1.Configuration, error) {
 
 	configs, err := client.ConfigurationV1alpha1().Configurations(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
 	if err != nil {
-		return nil, err
+		// This means that the Dapr Configurations CRD is not installed and there
+		// is are therefore no configuration items.
+		if serr, ok := err.(*apierrors.StatusError); ok &&
```
Comment (yaron2) [inline pkg/kubernetes/mtls.go:44]: See comment about "k8s.io/apimachinery/pkg/api/errors"

**Review by yaron2 — CHANGES_REQUESTED** [review]

**Review by artursouza — APPROVED** [review]
LGTM

File: pkg/kubernetes/components.go [hunk 5]
```diff
@@ -37,17 +37,14 @@ func PrintComponents(name, outputFormat string) error {
 		}
 
 		list, err := client.ComponentsV1alpha1().Components(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
-		if err != nil {
-			// This means that the Dapr Components CRD is not installed and there
-			// is are therefore no component items.
-			if serr, ok := err.(*apierrors.StatusError); ok &&
-				serr.ErrStatus.Reason == meta_v1.StatusReasonNotFound {
-				list = &v1alpha1.ComponentList{
```
Comment (yaron2) [inline pkg/kubernetes/components.go]: LGTM, there's no change needed but in general `Items` is a slice and as such doesn't need to be initialized with `[]v1alpha1.Component`

**Review by yaron2 — APPROVED** [review]

**Pull request merged**